xmtp_mls/migrations/2024-08-07-213816_create-private-preference-store/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "private_preferences"(
+    -- The inbox_id the private preferences are owned by
+    "inbox_id" text NOT NULL,
```

Comment:
We might not need an inbox_id since every item in this list will be for the person who is logged in they shouldn't be getting other peoples preferences.